web/src/main/webResources/WEB-INF/data/config/index/records.json

```diff
@@ -997,6 +997,32 @@
                   }
                 }
               },
+              "langfre": {
+                "type": "text",
+                "analyzer": "french",
+                "copy_to": [
+                  "anyfre"
+                ],
+                "fields": {
+                  "keyword": {
+                    "type": "keyword",
+                    "ignore_above": 256
+                  }
+                }
+              },
+              "langger": {
+                "type": "text",
+                "analyzer": "german",
+                "copy_to": [
+                  "anyfre"
+                ],
+                "fields": {
+                  "keyword": {
+                    "type": "keyword",
+                    "ignore_above": 256
+                  }
+                }
+              },
```

Comment:
Can you also add italian as we are going to experiment that in geocat.ch too probably in the short term.
And add some instructions in https://github.com/geonetwork/core-geonetwork/tree/main/es#multilingual-configuration-beta with link to Elasticsearch language analyzers.